backend/portal/models.py

```diff
@@ -48,17 +50,54 @@ class Content(models.Model):
     admin_comment = models.CharField(max_length=255, null=True, blank=True)
     target_populations = models.ManyToManyField(TargetPopulation, blank=True)
     priority = models.IntegerField(default=0)
+    creator = models.ForeignKey(User, on_delete=models.SET_NULL, null=True, blank=True)
 
     class Meta:
         abstract = True
 
+    def _get_email_subject(self):
```

Comment:
if these are meant to be private methods, they should start with two underscores, not one.

Reply:
hm
https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name

Reply:
damn u right, that's so weird, I've always assumed the name mangling is a good way to enforce private

Reply:
wait i didn't say i preferred single underscore. tbh i don't know which i prefer was hoping for u to choose one lol.

Reply:
lol i'd just stick with SO's opinion, single underscore for private methods, double underscore if you're concerned about inheritance overriding